modules/migrating-sccs-during-the-manual-upgrade.adoc

```diff
@@ -0,0 +1,351 @@
+  labels:
+     app.kubernetes.io/component: scanner
+     app.kubernetes.io/instance: stackrox-central-services
+     app.kubernetes.io/name: stackrox
+     app.kubernetes.io/part-of: stackrox-central-services
+     app.kubernetes.io/version: 4.4.0
+  name: use-scanner-scc
+  namespace: stackrox
+rules:
+- apiGroups:
+  - security.openshift.io
+  resourceNames:
+  - nonroot-v2
+  resources:
+  - securitycontextconstraints
+  verbs:
+  - use
+- - -
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding # <5>
+metadata:
+  annotations:
+     email: support@stackrox.com
+     owner: stackrox
+  labels:
+     app.kubernetes.io/component: central
+     app.kubernetes.io/instance: stackrox-central-services
+     app.kubernetes.io/name: stackrox
+     app.k ubernetes.io/part-of: stackrox-central-services
+     app.kubernetes.io/version: 4.4.0
+  name: central-db-use-scc # <6>
+  namespace: stackrox 
+roleRef: # <7>
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: use-central-db-scc
+subjects: # <8>
+- kind: ServiceAccount
+  name: central-db
+  namespace: stackrox
+- - -
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  annotations:
+     email: support@stackrox.com
+     owner: stackrox
+  labels:
+     app.kubernetes.io/component: central
+     app.kubernetes.io/instance: stackrox-central-services
+     app.kubernetes.io/name: stackrox
+     app.kubernetes.io/part-of: stackrox-central-services
+     app.kubernetes.io/version: 4.4.0
+  name: central-use-scc
+  namespace: stackrox
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: use-central-scc
+subjects:
+- kind: ServiceAccount
+  name: central
+  namespace: stackrox
+- - -
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  annotations:
+     email: support@stackrox.com
+     owner: stackrox
+  labels:
+     app.kubernetes.io/component: scanner
+     app.kubernetes.io/instance: stackrox-central-services
+     app.kubernetes.io/name: stackrox
+     app.kubernetes.io/part-of: stackrox-central-services
+     app.kubernetes.io/version: 4.4.0
+  name: scanner-use-scc
+  namespace: stackrox
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: use-scanner-scc
+subjects:
+- kind: ServiceAccount
+  name: scanner
+  namespace: stackrox
+- - -
+----
+<1> The type of Kubernetes resource, in this example, `Role`.
+<2> The name of the role resource.
+<3> The namespace in which the role is created. 
+<4> Describes the permissions granted by the role resource.
+<5> The type of Kubernetes resource, in this example, `RoleBinding`.
+<6> The name of the role binding resource.
+<7> Specifies the role to bind in the same namespace.
+<8> Specifies the subjects that are bound to the role.
+====
+
+.. Create the role and role binding resources specified in the `update-central.yaml` file by running the following command:
++
+[source,terminal]
+----
+$ oc -n stackrox create -f ./update-central.yaml
```

Comment:
I don't know whether we should specify that this needs to be run only on the cluster that runs the central workload. It might be self-explanatory, so I trust you to know whether we need to add this here